Tests/MetaTests/Visitors/TraversalVisitorTests.swift

```diff
@@ -0,0 +1,13 @@
+import XCTest
+import Meta
+import SnapshotTesting
+
+class TraversalVisitorTests: XCTestCase {
+    func testGeneratedTraversal() {
+        assertSnapshot(matching: URL.meta.fixtures.file(.article), as: .traversal)
+    }
+    
+    func testGeneratedTraversalForBowArrayK() {
```

Comment:
We should add tests for the others kind of Array: `NEA`, `NonEmptyArray`

Reply:
Just added a few cases of `NEA`, `NonEmptyArray` and `Array` in the fixtures, so the current tests already cover those cases properly.